qa/acceptance/spec/centos/lib/install_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../spec_helper'
+
+describe "artifacts", :platform => :centos do
+
+  shared_examples "installable" do |host, name|
+
+    before(:each) do
+      install("logstash-2.3.1-1.noarch.rpm", host)
+    end
+
+    it "is installed on #{name}" do
+      expect("logstash").to be_installed.on(host)
+    end
+
+    it "is running in #{name}" do
+      start_service("logstash", host)
+      expect("logstash").to be_running.on(host)
+      stop_service("logstash", host)
+    end
+
+    it "is removable on #{name}" do
+      uninstall("logstash", host)
+      expect("logstash").to be_removed.on(host)
+    end
+  end
+
+  config = ServiceTester.configuration
+  config.servers.each do |host|
+    it_behaves_like "installable", host, config.lookup[host]
+  end
+end
```

Comment:
I am usually in favor of explicit vs implicit, but I am wondering with all this `host` repetition.
Should we make the `host` implicit on the matcher? Our test will 99% of the time target one machine, the current one.

This would change the test to this format.
```ruby
it "is running in #{name}" do
 start_service("logstash")
 expect("logstash").to be_running
 stop_service("logstash")
end
```